frontend/src/app/reducers.ts

```diff
@@ -8,6 +9,13 @@ import projectsReducer, { defaultState as defaultProjectsState, ProjectsState }
 import { CustomWindow } from '../shared/custom.window';
 declare let window: CustomWindow;
 
+// Reducers
+const reducers = combineReducers({
+  auth: authReducer,
+  notifs: notifReducer,
```

Comment:
Sure

Reply:
I've looked into this and am now thinking it may be more trouble than it's worth.

`redux-notifications` expects the notifications to be found under then `notifs` key in the store:

https://github.com/indexiatech/re-notif/blob/master/src/components/Notifs.js#L84

We could overwrite the entire component however this would result in us pulling a lot more code into our codebase.